pkg/plugins/runtime/k8s/webhooks/validation.go

```diff
@@ -43,6 +44,10 @@ type validatingHandler struct {
 	decoder      admission.Decoder
 }
 
+var meshServiceSupportImplemented = map[core_model.ResourceType]bool{
```

Comment:
We can easily forget to add it here (other prs). Should we have some flag saying that it supports, or. a method returning supported policies?